vhdbuilder/packer/vhd-image-builder-arm64-gen2.json

```diff
@@ -609,15 +609,46 @@
     {
       "type": "file",
       "direction": "download",
-      "source": "/opt/azure/containers/image-bom.json",
-      "destination": "image-bom.json"
+      "source": "/opt/azure/containers/trivy-report.json",
+      "destination": "trivy-report.json"
     },
     {
       "type": "file",
       "direction": "download",
       "source": "/opt/azure/containers/trivy-images-table.txt",
       "destination": "trivy-images-table.txt"
     },
+    {
+      "type": "file",
+      "source": "vhdbuilder/packer/storage-scan.sh",
+      "destination": "/home/packer/storage-scan.sh"
```

Comment:
could we extend `trivy-scan.sh` to include both the trivy + storage scanning logic and potentially run them in parallel?

that way we simplify to:
- shell (execute)
- file download
- shell (rm dangling reports)